src/tools/miri/tests/pass-dep/concurrency/tls_pthread_drop_order.rs

```diff
@@ -5,6 +5,9 @@
 //! in `register_dtor_fallback`, which uses a *single* pthread_key to manage a thread-local list of
 //! dtors to call.
 
+// FIXME(obeis): Do not allow `static_mut_refs` lint, it's UB
```

Comment:
There's almost certainly no UB in this test, it's a Miri test -- Miri would find the UB.

The lint isn't saying "there's definite UB here", it says "this is sus, maybe write it in a way that's less error-prone".